src/routes/github/create-branch/github-create-branch-get.ts

```diff
@@ -27,7 +27,12 @@ export const GithubCreateBranchGet = async (req: Request, res: Response, next: N
 		return next(new Error(Errors.MISSING_ISSUE_KEY));
 	}
 
-	const subscriptions = await Subscription.getAllForHost(jiraHost);
+	let subscriptions;
+	if (gitHubAppConfig.gitHubAppId) {
+		subscriptions = await Subscription.getAllForHost(jiraHost, gitHubAppConfig.gitHubAppId);
```

Comment:
Will this return cloud values if we pass undefined for gutHubAppId?

Reply:
I agree, `getAllForHost` already handles the case for both GH cloud and GHE server. So, don't see much point for this `if-else`.

Reply:
This will return both cloud and server if we pass undefined for gitHubAppId